src/overlays/actors/ovl_Magic_Dark/z_magic_dark.c

```diff
@@ -79,6 +79,11 @@ void MagicDark_DiamondUpdate(Actor* thisx, PlayState* play) {
     if (1) {}
 
     // See `ACTOROVL_ALLOC_ABSOLUTE`
+    //! @bug This condition is too broad, the actor will also be killed by warp songs. But warp songs do not use an
+    //! actor which overlay uses `ACTOROVL_ALLOC_ABSOLUTE`. There is no reason to kill the actor in this case.
```

Comment:
I want to prevent confusion between the allocation of the actor instance and of the actor overlay

Is there a way in proper English to phrase this not weirdly?
> But warp songs do not use an actor which overlay uses `ACTOROVL_ALLOC_ABSOLUTE`.
> But warp songs do not use an actor which uses an overlay which uses `ACTOROVL_ALLOC_ABSOLUTE`.
> But warp songs do not use an actor that uses an overlay which uses `ACTOROVL_ALLOC_ABSOLUTE`.
> But warp songs do not use an actor that uses an overlay with `ACTOROVL_ALLOC_ABSOLUTE`.
> But warp songs do not use an actor that relies on an overlay with `ACTOROVL_ALLOC_ABSOLUTE`.

? (NotLikeThis)

Reply:
I feel like in this context, you dont need to be hyper specific about the mechanics of the overlay allocation.
If someone sees this and wants to learn more, they are going to read into how its actually working. And not to mention the define being referenced in the same setance specifies `ACTOROVL_`

I think "actor which uses" is fine, with it being implied that we are talking about the overlay of that actor